beetmoverscript/script.py

```diff
@@ -86,6 +95,46 @@
     await retry_download(context=context, url=source, path=abs_file_path)
     await retry_upload(context=context, destinations=destinations, path=abs_file_path)
 
+    if update_balrog_manifest:
+        context.balrog_manifest.append(
+            enrich_balrog_manifest(context, abs_file_path, locale, destinations)
+        )
+
+
+def enrich_balrog_manifest(context, path, locale, destinations):
+    props = context.properties
+
+    if props["branch"] == 'date':
+        # nightlies from dev branches don't upload to archive.m.o
+        url = "{prefix}/{s3_key}".format(prefix="http://bucketlister-delivery.stage.mozaws.net",
+                                         s3_key=destinations[0])
+        url_replacements = []
+    else:
+        # we extract the dated destination as the 'latest' is useless
+        url = "{prefix}/{s3_key}".format(prefix="https://archive.mozilla.org",
+                                         s3_key=destinations[0])
+        url_replacements = [['http://archive.mozilla.org/pub, http://download.cdn.mozilla.net/pub']]
+
+    return {
+        "tc_fennec_nightly": True,
```

Comment:
Added as a reminder in https://github.com/mozilla-releng/beetmoverscript/issues/12